From: Simon McVittie <smcv@debian.org>
Date: Sun, 6 Dec 2009 23:44:23 +0000
Subject: [PATCH] Use setuptools to get egg support (Debian #382272)

Bug-Debian: http://bugs.debian.org/382272
---
 setup.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/setup.py b/setup.py
index d2ba0d5..27d239b 100755
--- a/setup.py
+++ b/setup.py
@@ -6,7 +6,7 @@ import sys
 import os
 import glob
 try:
-    from distutils.core import setup, Command
+    from setuptools import setup, Command
     from distutils.command.build import build
     from distutils.command.build_py import build_py
     from distutils.command.install_data import install_data
-- 
