From: Simon McVittie <smcv@debian.org>
Date: Sun, 6 Dec 2009 23:44:30 +0000
Subject: [PATCH] rst.el: use Debian-style tool names, without the .py extension

Bug-Debian: http://bugs.debian.org/438667
---
 tools/editors/emacs/rst.el |   16 ++++++++--------
 1 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/tools/editors/emacs/rst.el b/tools/editors/emacs/rst.el
index 40213d6..a47943a 100644
--- a/tools/editors/emacs/rst.el
+++ b/tools/editors/emacs/rst.el
@@ -3201,13 +3201,13 @@ document with \\[rst-compile]."
   :version "21.1")
 
 (defvar rst-compile-toolsets
-  '((html . ("rst2html.py" ".html" nil))
-    (latex . ("rst2latex.py" ".tex" nil))
-    (newlatex . ("rst2newlatex.py" ".tex" nil))
-    (pseudoxml . ("rst2pseudoxml.py" ".xml" nil))
-    (xml . ("rst2xml.py" ".xml" nil))
-    (pdf . ("rst2pdf.py" ".pdf" nil))
-    (s5 . ("rst2s5.py" ".xml" nil)))
+  '((html . ("rst2html" ".html" nil))
+    (latex . ("rst2latex" ".tex" nil))
+    (newlatex . ("rst2newlatex" ".tex" nil))
+    (pseudoxml . ("rst2pseudoxml" ".xml" nil))
+    (xml . ("rst2xml" ".xml" nil))
+    (pdf . ("rst2pdf" ".pdf" nil))
+    (s5 . ("rst2s5" ".xml" nil)))
   "An association list of the toolset to a list of the (command to use,
 extension of produced filename, options to the tool (nil or a
 string)) to be used for converting the document.")
@@ -3215,7 +3215,7 @@ string)) to be used for converting the document.")
 ;; Note for Python programmers not familiar with association lists: you can set
 ;; values in an alists like this, e.g. :
 ;; (setcdr (assq 'html rst-compile-toolsets)
-;;      '("rst2html.py" ".htm" "--stylesheet=/docutils.css"))
+;;      '("rst2html" ".htm" "--stylesheet=/docutils.css"))
 
 
 (defvar rst-compile-primary-toolset 'html
-- 
