--- __init__.py	2009-04-01 22:00:21.000000000 +0200
+++ /usr/local/lib/python2.6/dist-packages/docutils-0.6-py2.6.egg/docutils/writers/s5_html/__init__.py	2010-07-26 09:56:32.145333990 +0200
@@ -315,11 +315,10 @@ class S5HTMLTranslator(html4css1.HTMLTra
         self.s5_header.extend(header)
 
     def visit_section(self, node):
-        if not self.section_count:
-            self.body.append('\n</div>\n')
+        self.body.append('\n</div>\n')
         self.section_count += 1
         self.section_level += 1
-        if self.section_level > 1:
+        if self.section_level > 2:
             # dummy for matching div's
             self.body.append(self.starttag(node, 'div', CLASS='section'))
         else:

